src/app/initialisation/syncPoolService.js

```diff
@@ -49,9 +49,9 @@ function SyncPoolService($q, mediator, syncService) {
 
     //Initialisation of sync data sets to manage.
     return $q.all([
-      syncService.manage(config.datasetIds.workorders, {}, {filter: filter}, config.syncOptions),
-      syncService.manage(config.datasetIds.workflows, {}, {}, config.syncOptions),
-      syncService.manage(config.datasetIds.results, {}, {}, config.syncOptions)
+      syncService.manage(config.datasetIds.workorders, config.syncOptions.workorders, {filter: filter}, {}),
```

Comment:
@witmicko Do you know the reason why this is initialized in mobile app rather than in each module itself. 
It looks like we leaking here some implementation's and configuration that should be done in modules.